store/copr/coprocessor.go

```diff
@@ -1312,6 +1409,24 @@ func (worker *copIteratorWorker) handleBatchCopResponse(bo *Backoffer, batchResp
 		// TODO: check OOM
 		worker.sendToRespCh(resp, ch, false)
 	}
+	for _, t := range tasks {
+		task := t.task
+		// when the error is generated by client, response is empty, skip warning for this case.
+		if len(batchResps) != 0 {
+			firstRangeStartKey := task.ranges.At(0).StartKey
+			lastRangeEndKey := task.ranges.At(task.ranges.Len() - 1).EndKey
+			logutil.Logger(bo.GetCtx()).Error("response of batched task missing",
+				zap.Uint64("id", task.taskID),
+				zap.Uint64("txnStartTS", worker.req.StartTs),
+				zap.Uint64("regionID", task.region.GetID()),
+				zap.Uint64("bucketsVer", task.bucketsVer),
+				zap.Int("rangeNums", task.ranges.Len()),
+				zap.ByteString("firstRangeStartKey", firstRangeStartKey),
+				zap.ByteString("lastRangeEndKey", lastRangeEndKey),
```

Comment:
What's the meaning of printing the start key of the first range and the end key of the last range...?
They don't belong to the same range and there are also holes between them...

Reply:
It's the start key and end key for one batched task(it might be in the same region), I just copy the log from [here](https://github.com/pingcap/tidb/blob/760dbe102d5b28967f47bed0e361a4c9d2cb461f/store/copr/coprocessor.go#L1155-L1164).

Reply:
okay..